frontend/front/src/components/SurveyComponent/HeatPumpPhoneField.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Just split the off from `SurveyComponent` to reduce the amount of the things happening in there, since its getting pretty big.